debian/control

```diff
@@ -12,7 +12,8 @@ Vcs-Browser: https://github.com/dlech/keepasshttp
 
 Package: keepass2-plugin-keepasshttp
 Architecture: all
-Depends: ${shlibs:Depends}, ${misc:Depends}
+Depends: ${shlibs:Depends}, ${misc:Depends}, libmono-system-xml-linq4.0-cil, libmono-system-data-datasetextensions4.0-cil, libmono-system-runtime-serialization4.0-cil, libnewtonsoft-json5.0-cil
```

Comment:
I think just adding `${cli:Depends}` here should work instead of specifying the individual packages.